src/st_stuff.c

```diff
@@ -86,1400 +84,1712 @@
+{
+    int lumpnum = W_CheckNumForName(name);
+    if (lumpnum < 0)
+    {
+        lumpnum = (W_CheckNumForName)(name, ns_sprites);
+        if (lumpnum < 0)
+        {
+            return NULL;
+        }
+    }
+    return V_CachePatchNum(lumpnum, PU_STATIC);
+}
 
-// weapon ownership patches
-static patch_t *arms[6][2];
+static void LoadFacePatches(void)
+{
+    char lump[9] = {0};
 
-// ready-weapon widget
-static st_number_t w_ready;
+    for (int painface = 0; painface < ST_NUMPAINFACES; ++painface)
+    {
+        for (int straightface = 0; straightface < ST_NUMSTRAIGHTFACES;
+             ++straightface)
+        {
+            M_snprintf(lump, sizeof(lump), "STFST%d%d", painface, straightface);
+            array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
+        }
 
-// [Alaux]
-static boolean hud_animated_counts;
-int st_health = 100;
-int st_armor = 0;
+        M_snprintf(lump, sizeof(lump), "STFTR%d0", painface); // turn right
+        array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
 
-//jff 2/16/98 status color change levels
-int ammo_red;      // ammo percent less than which status is red
-int ammo_yellow;   // ammo percent less is yellow more green
-int health_red;    // health amount less than which status is red
-int health_yellow; // health amount less than which status is yellow
-int health_green;  // health amount above is blue, below is green
-int armor_red;     // armor amount less than which status is red
-int armor_yellow;  // armor amount less than which status is yellow
-int armor_green;   // armor amount above is blue, below is green
+        M_snprintf(lump, sizeof(lump), "STFTL%d0", painface); // turn left
+        array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
 
-boolean hud_backpack_thresholds; // backpack changes thresholds
-boolean hud_armor_type; // color of armor depends on type
+        M_snprintf(lump, sizeof(lump), "STFOUCH%d", painface); // ouch!
+        array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
 
- // in deathmatch only, summary of frags stats
-static st_number_t w_frags;
+        M_snprintf(lump, sizeof(lump), "STFEVL%d", painface); // evil grin ;)
+        array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
 
-// health widget
-static st_percent_t w_health;
+        M_snprintf(lump, sizeof(lump), "STFKILL%d", painface); // pissed off
+        array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
+    }
 
-// weapon ownership widgets
-static st_multicon_t w_arms[6];
+    M_snprintf(lump, sizeof(lump), "STFGOD0");
+    array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
 
-// face status widget
-static st_multicon_t w_faces;
+    M_snprintf(lump, sizeof(lump), "STFDEAD0");
+    array_push(facepatches, V_CachePatchName(lump, PU_STATIC));
 
-// keycard widgets
-static st_multicon_t w_keyboxes[3];
+    // [FG] support face gib animations as in the 3DO/Jaguar/PSX ports
+    int painface;
```

Comment:
Let's have just one variable of that name in the function context.

Reply:
FIxed in https://github.com/fabiangreffrath/woof/pull/1916/commits/6f49928acaec7ad5661e729e557502b2977f0d82